appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
Sorry to be dense, but I still don't get it.  what is the different between the second block and the third block?  That is, the second block says "set button click handler for [component} to ..."   and the third block says
"when there is a button.click on [component] do ..."
What's the difference between these two things?